lib/genre.rb

```diff
@@ -12,4 +12,8 @@ def add_item(item)
     @items.push(item) unless @items.include?(item)
     item.add_genre(self)
   end
+
+  # def to_s
+  #   "Name of Genre: #{@name}"
+  # end
```

Comment:
Don't left comented code, try to clean it before do the PR. That will help us to keep a clean code in the dev branch.